python-sdk/tests_integration/test_example_dags.py

```diff
@@ -83,15 +85,14 @@ def order(dag_id: str) -> int:
     return -1
 
 
-dag_bag = get_dag_bag()
+@pytest.fixture()
+def example_dag_ids():
+    dag_bag: DagBag = get_dag_bag()
+    for dag_id in sorted(dag_bag.dag_ids, key=order):
+        yield dag_id, dag_bag
```

Comment:
I think we can make `dag_bag` its own fixture. WDYT?

Reply:
@feluelle We can, but are you also suggesting using it individually in some tests?

Reply:
Yes so that you don't have to return it there.

Instead, what you can do then is the following:
```python
@pytest.fixture(scope="module")
def dag_bag() -> DagBag:
    return get_dag_bag()


@pytest.fixture()
def example_dag_ids(dag_bag) -> str.
    for dag_id in sorted(dag_bag.dag_ids, key=order):
        yield dag_id


def test_example_dag(session, example_dag_ids, dag_bag):
    ...
```

Reply:
Got it, thanks! 👍

Reply:
@feluelle the existing code is raising an issue since there can only be one yield statement in a fixture. I have taken new approach to this, please take a look.

Reply:
Resolved this on a call, and pushed agreed-upon changes.